index.js

```diff
@@ -0,0 +1,42 @@
+//select the tv button element.
+//create eventListener to initiate function when tv button is clicked.
+document.querySelector("#new-list").addEventListener("click", createList)
+
+//function to create a new list onclick.
+function createList() {
+
+    //create outer div to contain list.
+    const newList = document.createElement('div')
+    newList.classList.add("list-container")
+   
+
+    //create inner div for input and button element.
+    const inputDiv = document.createElement('div')
+    inputDiv.classList.add("list-input")
```

Comment:
I see so the classList.add() adds a new one every time the user types?

This is a great way at addressing the task 🙌 🐰 ✨